public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -84,8 +84,6 @@ const normalizedPanel = (panel: CustomPanelType): CustomPanelType => ({
 
 export const selectPanelList = (rootState): CustomPanelType[] => rootState.customPanel.panelList;
 
-const {setToast} = useToast();
```

Comment:
linter says "useToast" can't be called at top-level.
Integrated into each method using it.